docker/Dockerfile

```diff
@@ -291,6 +303,8 @@ RUN if [ -n "$CUDA_VERSION" ] ; then \
 # Install Flash Attention
 ##########################
 RUN if [ -n "$CUDA_VERSION" ] ; then \
+        pip${PYTHON_VERSION} install --upgrade --no-cache-dir ninja==1.11.1 && \
+        pip${PYTHON_VERSION} install --upgrade --no-cache-dir --force-reinstall packaging==22.0 && \
```

Comment:
do you know why packaging needs a force re-install? It's not an issue, just curious

Reply:
packaging might be installed already above if apex is installed; in that case to avoid an error we just force a reinstall to a pinned version we know works.